### PR TITLE
add nameserver for resolution in cloud pritunl

### PR DESCRIPTION
**Pull request by angeloskyratzakos**
#### Summary

- Add nameserver in resolv.conf
- minor fixes

### CODE REVIEW

**Comment by gigawhitlocks** [comment]
I run Linux and don't use Pritunl so I'm going to swap my review for Gabe

**Comment by gigawhitlocks** [comment]
also ha-ha, **my** VPN client works in split tunnel mode :joy:

**Comment by gigawhitlocks** [comment]
Well, I see your :confused: reaction, so I'll include a screenshot -- I _think_ this check box is split tunnel mode:
Screenshot from 2020-07-28 11-10-34

But maybe it's not

File: site/content/internal/infrastructure/vpn-cloud.md [hunk 1]
```diff
@@ -37,12 +37,15 @@ Select `Show More` and hit `Download Profile`
...
 
 - Go to Networking tab and update the DNS settings. 
-   * Select for the Mode to be `Split DNS`.  
+   * Select for the Mode to be `Full DNS`.  
    * As `Servers` put: `pritunl.core.cloud.mattermost.com` which is VPN's server IP.
    * In `Domains` put: `cloud.mattermost.com`, this will split traffic for those domains
     <span style="display:block;text-align:center;width:50%">![Network settings Viscosity](/img/vpn_cloud_6.jpg)</span>
+    
+6. Add, if it is not there, in your `/etc/resolv.conf`:
+    `nameserver 10.247.0.2`
```
Comment (gabrieljackson) [inline site/content/internal/infrastructure/vpn-cloud.md:46]: I haven't done this and don't have any issues, but I assume something must require it.

Comment (angeloskyratzakos) [reply site/content/internal/infrastructure/vpn-cloud.md:46]: It was failing on Eli's configuration, so I was thinking it's a good addition

**Pull request merged**